skimage/exposure/exposure.py

```diff
@@ -346,7 +345,56 @@ def cumulative_distribution(image, nbins=256):
     return img_cdf, bin_centers
 
 
-def equalize_hist(image, nbins=256, mask=None):
+def _equalize_hist_uint(image, mask=None, lut_max=255):
```

Comment:
`lut_max` is not documented :slightly_smiling_face:

Reply:
now fixed (forgot to come back to this PR!)